package/utils/busybox/Makefile

```diff
@@ -77,13 +77,24 @@ define Package/busybox/config
 endef
 
 ifdef CONFIG_BUSYBOX_CONFIG_FEATURE_SYSLOG
-define Package/busybox/conffiles
+define Package/busybox/conffiles/syslog
 /etc/syslog.conf
 endef
+endif
 
-Package/busybox-selinux/conffiles = $(Package/busybox/conffiles)
+ifneq ($(CONFIG_BUSYBOX_$(BUSYBOX_SYM)_CROND),)
+define Package/busybox/conffiles/crond
+/etc/crontabs/
+endef
 endif
 
+define Package/busybox/conffiles
+$(Package/busybox/conffiles/syslog)
+$(Package/busybox/conffiles/crond)
```

Comment:
I'm currently trying to debug this but can't really figure it out. `/etc/syslog.conf` is created but under no circumstances is `/etc/syslog.conf` added to `/lib/keep.d/`

Reply:
> I'm currently trying to debug this but can't really figure it out. `/etc/syslog.conf` is created but under no circumstances is `/etc/syslog.conf` added to `/lib/keep.d/`

It goes into `CONTROL/conffiles` I believe, and `opkg` puts it into the `conffiles` section of `/usr/lib/opkg/status`.  Are you seeing it there?